src/helper.js

```diff
@@ -520,6 +526,21 @@ function createProperty(property = {}) {
   return undefined;
 }
 
+function getIpAddress(request) {
+  const IP_ADDRESS = /\b(?:\d{1,3}\.){3}\d{1,3}\b/g;
```

Comment:
Let's change it to a regex with strict IPv4 and IPv6 validation, see https://git.corp.adobe.com/anischev/openapi-specs/blob/TNT-36261/delivery/components/Geo.yaml#L9